FIAT/reference_element.py

```diff
@@ -482,6 +483,18 @@ def __init__(self):
         topology = {0: {0: (0,)}}
         super(Point, self).__init__(POINT, verts, topology)
 
+    def get_facet_element(self):
+        raise ValueError("A point does not have any facets.")
```

Comment:
You could have probably left this method out, but whatever...

Reply:
You're right. I thought this would give a confusing `NotImplementedError("Should be implemented in a subclass")` but it wont. Removed in 9117dbdf5fff94dcf3a0117d3fbfdbf11f0d7df9